test/e2e/cluster_config.sh

```diff
@@ -69,6 +69,8 @@ clusters:
     profile: ${WORKER_PROFILE}-splitaz
     min_size: 3
     max_size: 21
+    config_items:
+      cpu_manager_policy: static
   - discount_strategy: spot_max_price
     instance_types: ["m4.large", "m5.large", "m5.xlarge", "m4.xlarge"]
```

Comment:
Does it mean that the cpu_manager_policy can be set only in the node pools with these type of instances?